### PR TITLE
hide error symbol from being printed

### PR DESCRIPTION
**Pull request by CatchMe2**
## Changes

This PR prevents the error symbol from being printed.
console.log output of the previous implementation:
image

console.log output of the new implementation
image

## Checklist

- [x] Apply one of following labels; `major`, `minor`, `patch` or `skip-release`
- [x] I've updated the documentation, or no changes were necessary
- [x] I've updated the tests, or no changes were necessary

### CODE REVIEW

**Comment by drdaemos** [comment]
Is there is a specific reason to this change? 
e.g. why don’t we want to have this symbol printed?

File: src/errors/InternalError.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { isError } from '../utils/typeUtils'
+import { isNativeError } from 'node:util/types'
```
Comment (kibertoad) [inline src/errors/InternalError.ts:1]: why this change? this method was deprecated AFAIR

Comment (CatchMe2) [reply src/errors/InternalError.ts:1]: It's not depreciated: https://nodejs.org/api/util.html#utiltypesisnativeerrorvalue
I think that it's better to use std lib solutions

**Comment by CatchMe2** [comment]
> Is there is a specific reason to this change? e.g. why don’t we want to have this symbol printed?

This symbol was meant to be internal, used for instance check only. It doesn't provide any valuable info.

File: src/errors/InternalError.ts (hunk 1 above)
Comment (kibertoad) [reply src/errors/InternalError.ts:1]: ok, then it makes sense

**Comment by CatchMe2** [comment]
After a discussion, we decided to leave printing it as before.

**Review by drdaemos — APPROVED** [review]

**Review by kibertoad — APPROVED** [review]
